tests/ApsMapper/FindByPidProgrammeMapperTest.php

```diff
@@ -264,6 +264,7 @@ public function testMappingClip()
             1103,
             MediaTypeEnum::VIDEO,
             1201,
+            0,
```

Comment:
Why does this need updating but no other bit?

Reply:
AggregatedGalleriesCount is new for Clips, in other cases it has been renamed from AvailableGalleriesCount to AggregatedGalleriesCount